src/styles.css

```diff
@@ -1,3 +1,9 @@
 body {
   font-family: sans-serif;
 }
+
+article {
+  border: 2px solid black;
```

Comment:
🙀 This is not good user experience. Please make the card hotpink!

Reply:
Good catch! Thanks, fixed it! :)